lib/openlogi/shipment.rb

```diff
@@ -0,0 +1,27 @@
+require "openlogi/base_object"
+
+module Openlogi
+  class Shipment < BaseObject
+    attribute :id, String
+    attribute :identifier, String
+    attribute :created_at, DateTime
```

Comment:
Ah `id` is not only a system value, it is used in requests. `created_at` is probably just the rails timestamp, maybe not intended to be public.